### PR TITLE
Letter signposting

### PR DESCRIPTION
**Pull request by kosiakkatrina**
**What**  
Add an option to send summary by letter.

Add a different note content for the letter.
Change the wording from email to message.
Open the summary dropdown when view summary message is clicked.

image

**Why**  
To make summary sharing more accessible to everyone.

Ticket: [BC2-10](https://hackney.atlassian.net/browse/BC2-10?atlOrigin=eyJpIjoiYTk2ZmI4NWRlMmNhNDhjNWFiY2QzMjVmMDUxYjVmODEiLCJwIjoiaiJ9)

### CODE REVIEW

File: components/Feature/SupportSummary/index.js [hunk 1]
```diff
@@ -222,15 +223,66 @@ const SupportSummary = ({
...
+                <TextArea
+                  value={emailBody}
+                  label="Add a note for the resident"
+                  name="support-summary-note"
+                  rows="20"
+                  onChange={value => {
+                    setEmailBody(value);
+                  }}
+                />
+              </div>
```
Comment (kosiakkatrina) [inline components/Feature/SupportSummary/index.js:285]: This is the same as the lines that appear to be removed in this file (225-233) except wrapped in a div for formatting

File: components/Feature/SupportSummary/index.js [hunk 2]
```diff
@@ -222,15 +223,66 @@ const SupportSummary = ({
...
               />
-              <TextArea
-                value={emailBody}
-                label="Add a note for the resident"
-                name="support-summary-note"
-                rows="20"
-                onChange={value => {
-                  setEmailBody(value);
-                }}
-              />
```
Comment (kosiakkatrina) [inline components/Feature/SupportSummary/index.js:233]: Doesn't actually get removed (see the comment below)

File: components/Form/Details/index.js [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
+    className={`govuk-details ${css['lbh-details']}`}
+    data-module="govuk-details"
+    id={`details-${id}`}>
     <summary
       className={`govuk-details__summary ${css['lbh-details__summary']}`}
       style={color && { color: `${color}` }}
       id={`summary-${id}`}
       onClick={onclick}>
-      <span className="govuk-details__summary-text">{title}</span>
+      <span id={`span-${id}`} className="govuk-details__summary-text">
```
Comment (kosiakkatrina) [inline components/Form/Details/index.js:13]: This is the only line that's changed (added an id to the span) the rest is just formatting

File: lib/use-cases/create-conversation.js [hunk 4]
```diff
@@ -1,23 +1,41 @@
...
 
-    const signPostingEmail = await this.sendSignpostEmail.execute({
-      emailAddress: resident.email,
-      signPostingMessage
-    });
-
-    if (signPostingEmail.response.status > 201) {
-      errors.push('Failed to send sign posting email to resident');
+    let signPostingEmail;
+    if (sharingMethod == 'email') {
```
Comment (owenpigott) [inline lib/use-cases/create-conversation.js:19]: I'd suggest enumerating this and other instances of letter/email, similar to the constants for Analytics events. Although less beneficial for Javascript over a typed language.

**Pull request merged**